app/src/main/java/com/door43/translationstudio/core/ImportUsfm.java

```diff
@@ -1087,11 +1087,14 @@ private boolean breakUpChapter(CharSequence text, String currentChapterStr) {
                 return false;
             }
         } else { // save stuff before first chapter
-            String chapter0 = "00"; // chapter "00" folder contains stuff that applies to the whole book, like title
-            success = saveSection("front", "intro", cleanedString);
-            successOverall = successOverall && success;
-            success = saveSection(chapter0, "title", mBookName);
-            successOverall = successOverall && success;
+            String strippedUsfmFrontTags = removeKnownUsfmTags(cleanedString);
+            if(strippedUsfmFrontTags.length() > 0) {
+                String chapter0 = "00"; // chapter "00" folder contains stuff that applies to the whole book, like title
+                success = saveSection("front", "intro", strippedUsfmFrontTags);
+                successOverall = successOverall && success;
+                success = saveSection(chapter0, "title", mBookName);
```

Comment:
fyi. there is migration to move away from the `00` dir we just never finished it. I finished switching to the new `front` dir in https://github.com/unfoldingWord-dev/ts-android/commit/b3efeea8ae7f882c6f118cd1d7d278a204960e9c. 

> Note: any translations created in the past dev builds should be deleted to avoid corrupt data. This intermediate migration was never released so we don't need to worry about users.